arch/risc-v/src/common/riscv_cpustart.c

```diff
@@ -38,8 +38,21 @@
 #include "sched/sched.h"
 #include "init/init.h"
 #include "riscv_internal.h"
+
+#ifdef CONFIG_BUILD_KERNEL
+#  include "riscv_mmu.h"
+#endif
+
 #include "chip.h"
 
+/****************************************************************************
+ * Public Data
+ ****************************************************************************/
+
+#ifdef CONFIG_BUILD_KERNEL
+extern uintptr_t g_kernel_pgt_pbase;
```

Comment:
should we put to the riscv_internal.h

Reply:
@xiaoxiang781216 
I think `riscv_mmu.h` might be better.

Reply:
Done.